hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -759,6 +759,8 @@ public FileStatus getFileStatus(Path f) throws IOException {
       if (ex.getResult().equals(OMException.ResultCodes.KEY_NOT_FOUND)) {
         throw new FileNotFoundException("File not found. path:" + f);
       }
+      LOG.debug("GetFileStatus failed for path {}", f, ex);
```

Comment:
perhaps error or warn log level is better than debug for this kind of situation?

Reply:
makes sense, I will change it to WARN